app/components/c-attribute-grid.vue

```diff
@@ -0,0 +1,176 @@
+    </div>
+    <div :class="b('toggle', { open: isOpen })" role="button" @click="toggle()">
+      <e-icon :class="b('arrow')" inline icon="i-arrow-down"/>
+    </div>
+  </div>
+</template>
+
+<script>
+  import cAttributeGridItems from '@/components/c-attribute-grid-items';
+
+  export default {
+    name: 'c-attribute-grid',
+    components: {
+      cAttributeGridItems,
+    },
+    // mixins: [],
+
+    props: {
+      /**
+       * Defines the items displayed in the attribute grid.
+       */
+      attributes: {
+        type: Array,
+        required: true,
+      },
+
+      /**
+       * Defines whether the grid should shrink on mobile or not.
+       */
+      shrinkOnMobile: {
+        type: Boolean,
+        default: false,
+      },
+    },
+
+    data() {
+      return {
+        isOpen: true,
+        openHeight: 0,
+        closedHeight: 0,
+      };
+    },
+
+    computed: {
+      modifiers() {
+        return {
+          shrinkOnMobile: this.shrinkOnMobile,
+          dontShrinkOnMobile: !this.shrinkOnMobile
+        };
+      },
+      maxHeight() {
+        if (!this.shrinkOnMobile) {
+          return 'inherit';
+        }
+
+        if (this.isOpen) {
+          return this.openHeight;
+        }
+
+        return this.closedHeight;
+      },
+    },
+    // watch: {},
+
+    // beforeCreate() {},
+    // created() {},
+    // beforeMount() {},
+
+    mounted() {
+      this.recalculateOpenHeight();
+      window.addEventListener('resize', this.recalculateOpenHeight);
+
+      this.recalculateClosedHeight();
+      window.addEventListener('resize', this.recalculateClosedHeight);
+
+      this.isOpen = false;
+    },
+    // beforeUpdate() {},
+    // updated() {},
+    // activated() {},
+    // deactivated() {},
+    // beforeDestroy() {},
+    // destroyed() {},
+
+    methods: {
+      toggle() {
+        this.isOpen = !this.isOpen;
+      },
+      recalculateOpenHeight() {
+        let openHeight = this.$el.querySelector('dl').clientHeight;
+
+        if (openHeight > 0) {
+          openHeight = `${openHeight}px`;
+        }
+
+        this.openHeight = openHeight;
+      },
+      recalculateClosedHeight() {
+        const dtHeight = this.$el.querySelector('dt').clientHeight;
+        const ddHeight = this.$el.querySelector('dd').clientHeight;
+        let closedHeight = dtHeight >= ddHeight ? dtHeight : ddHeight;
+
+        if (closedHeight > 0) {
+          closedHeight = `${closedHeight}px`;
+        }
+
+        this.closedHeight = closedHeight;
+      },
+    },
+
+    // render(createElement) {},
+  };
+</script>
+
+<style lang="scss">
+  .c-attribute-grid {
+    $toggle-animation-duration: 0.7s;
+
+    &__toggle {
+      display: none;
+      margin: $spacing--0 $spacing--10;
+      text-align: center;
+      border-top: thin solid $color-grayscale--600;
+    }
+
+    &--shrink-on-mobile &__toggle {
+      display: block;
+      padding: $spacing--5 $spacing--0 $spacing--0 $spacing--0;
```

Comment:
- [ ] Here you can just define `padding-top: $spacing--5;` instead of all 0 values around